src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -462,7 +464,7 @@ function EmojiPickerMenu(props) {
                 />
             );
         },
-        [isUsingKeyboardMovement, highlightedIndex, onEmojiSelected, preferredSkinTone, translate, highlightFirstEmoji],
+        [preferredSkinTone, highlightedIndex, isUsingKeyboardMovement, highlightFirstEmoji, styles.emojiHeaderContainer, styles.textLabelSupporting, translate, onEmojiSelected],
```

Comment:
NAB/queestion: why did you change the order? 😃

Reply:
wasn't me, Mr. Prettier did

Reply:
😃 ok then